lib/msal-browser/src/controllers/StandardController.ts

```diff
@@ -396,6 +400,37 @@ export class StandardController implements IController {
         });
     }
 
+    protected async setPlatformAuthProvider(
```

Comment:
This still looks like a reimplementation of `isPlatformBrokerAvailable`. I would suggest:
1. Moving this next to `isPlatformBrokerAvailable` and renaming to `getPlatformAuthProvider` returning a provider || null
2. In `isPlatformBrokerAvailable` just return `!!getPlatformAuthProvider()`
3. Above set `this.platformAuthProvider = getPlatformAuthProvider`